Makefile

```diff
@@ -0,0 +1,18 @@
+lint: bandit flake8 isort
+	
+test: coveralls 
```

Comment:
There are now two `test` commands in this file, which I don't think will work. I'd remove this one, as the `coveralls` command is sufficient.